src/components/detailedInfo/DetailedInfo.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom'; 
+import api from '../../utils/api/api';
+
+import { movieDetails } from '../../constants/movieDetails'; 
+import './detailedInfo.css';
+
+
+const DetailedInfo = () => {
+  const { movieId } = useParams();
+  let [movie, setMovie] = useState(null);
+  let [error, setError] = useState(null);
+  const [loading, setLoading] = useState(true); 
+
+  const fetchMovieData = async (movieID = '', movieName = '') => {
```

Comment:
Since fetchMovies is being used in multiple places. Lets create a common function in api folder and use that.